lib/gstreamer/va/vpp.py

```diff
@@ -17,8 +17,11 @@
 class VppTest(BaseVppTest):
   def before(self):
     super().before()
-    # TODO: need gst-va to fix target render device
-    vars(self).update(gstvpp = "vapostproc")
+    renderDevice = get_media().render_device.split('/')[-1]
+    if renderDevice != 'renderD128':
+      vars(self).update(gstvpp = f"va{renderDevice}postproc")
+    else:
+      vars(self).update(gstvpp = f"vapostproc")
```

Comment:
This should be ok since we have not refactored Vpp like the Encoder/Decoder.

However, you missed the `vacompositor` element in `get_vpp_opts`.